src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -423,13 +423,13 @@ class LedgerMasterImp
                 for (auto const& it : mHeldTransactions)
                 {
                     ApplyFlags flags = tapNONE;
-                    if (app_.getHashRouter().addSuppressionFlags (
-                            it.first.getTXID (), SF_SIGGOOD))
-                        flags = flags | tapNO_CHECK_SIGN;
```

Comment:
I'm leaving this instance out because when the Transactor calls checkValidity, the same flags will get checked, so this instance is redundant.